source/text/implementation/vss-strings-cursors.adb

```diff
@@ -276,7 +276,7 @@ package body VSS.Strings.Cursors is
 
    overriding procedure Invalidate (Self : in out Character_Cursor_Base) is
    begin
-      Self.Position := (1, 0, 0);
+      Self.Position := (others => <>);
```

Comment:
looks like a `null;` statement

Reply:
It is initialization to default values if they are defined. By default objects of Cursor type are initialized to invalid state.